### PR TITLE
implement several account api and cli 

### PR DESCRIPTION
**Pull request by baichuan3**
This PR contains the following implementations:

1, merge jsonrpc implementation for jsonrpsee RPC framework
2, support register jsonrpc methods for multi modules
3, implements several account api and cli

### CODE REVIEW

**Comment by jolestar** [comment]
@wow-sven @baichuan3  We need to think about how to keep the account. 

1. Implemention the account manager on the node server side, cli call the account API to manage the account, but the account API only be accessed through the local IP by default. Like ethereum and starcoin, this approach is suited for the P2P node network.
2. Implemention the account manager on the cli client side, and maintain the account in a local config file. 

I prefer the first approach because the Rooch node needs an account do some operation.

File: crates/rooch/src/account/mod.rs
Comment (jolestar) [inline crates/rooch/src/account/mod.rs:1]: This file should be at `src/commands/account/mod.rs` ?

**Review by jolestar — APPROVED** [review]

**Comment by baichuan3** [comment]
> @wow-sven @baichuan3 We need to think about how to keep the account.
> 
> 1. Implemention the account manager on the node server side, cli call the account API to manage the account, but the account API only be accessed through the local IP by default. Like ethereum and starcoin, this approach is suited for the P2P node network.
> 2. Implemention the account manager on the cli client side, and maintain the account in a local config file.
> 
> I prefer the first approach because the Rooch node needs an account do some operation.

The current implementation is based on the first approach, but it needs to add local IP restrictions

File: crates/rooch/src/account/mod.rs
Comment (baichuan3) [reply crates/rooch/src/account/mod.rs:1]: fixed

**Pull request merged**